src/libs/conduit/conduit_data_type.hpp

```diff
@@ -380,6 +380,7 @@ class CONDUIT_API DataType
 //-----------------------------------------------------------------------------
     index_t     id()    const { return m_id;}
     std::string name()  const { return id_to_name(m_id);}
+    std::string group() const { return id_to_group(m_id); }
 
```

Comment:
Do we need a new "group" concept to be in the public interface?

We already have:

    bool        is_number()           const;
    bool        is_floating_point()   const;
    bool        is_integer()          const;
    bool        is_signed_integer()   const;
    bool        is_unsigned_integer() const;

It seems the group "strings" kind of capture this, can these be used?

Reply:
As we discussed earlier, I'm not thrilled with the `group` concept either. I'll remove it for the time being, but I do think some general "these are trying to represent exactly the same data with different bitwidths" concept would be useful.